.circleci/config.yml

```diff
@@ -209,26 +209,33 @@ commands:
             python3 ./tools/dependency_summary.py --all-android-targets --package megazord --format pom --check megazords/full/android/dependency-licenses.xml
   bench-all:
     steps:
-      - test-setup
       - run:
           name: Build and Run Benchmarks
           command: cargo bench --all
   restore-sccache-cache:
     steps:
       - restore_cache:
           name: Restore sccache cache
-          key: sccache-cache-stable-{{ arch }}-{{ .Environment.CIRCLE_JOB }}
+          keys: 
+          # We have multiple keys to increase the chance of a cache hit
+          # in case the Cargo.lock is updated, we still want to retrieve
+          # some cache
+            - sccache-cache-stable-{{ arch }}-{{ checksum "Cargo.lock" }}
```

Comment:
I think I should bring back the `--{ .Environment.CIRCLE_JOB }}` and have it so jobs of the same **type** share caches.. because otherwise, the fastest job (which wouldn't doesn't need too much cache) would upload the cache for that checksum and every other job is stuck with it and won't update it